website-migration.rst

```diff
@@ -0,0 +1,61 @@
+.. post:: Mar 3, 2023
+   :tags: website, migration 
+   :category: Changelog
+
+.. Categories: https://docs.google.com/document/d/1io3HN61wpFtb6XP-cyONhV5CNWiZ5dbhZM4nvifyT5g/edit#heading=h.3xtrxdnntzqi
+
+Read the Docs website migration to about.readthedocs.com
+========================================================
```

Comment:
This is a bit of a different topic? That seems like a marketing post about the new site, where as this is a changelog entry explaining the timeline.

Reply:
I would say that the former headline gives the reader an impression that it's going to be a technical blog post unveiling techniques used for a migration. But I think we're rather trying to avoid becoming technical? We don't mention Pelican or DNS redirects? (I'm yet to read to the finish, sorry I was a bit distracted now and before)